UnityProject/Assets/Scripts/Health/Objects/DamageWeaknesses.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class DamageWeaknesses
```

Comment:
couldn't this be called ``DamageResistance`` since that means you could be more resistant or less resistant (aka weak) against a particular damage type.

This also means we should remove the natural armour body parts have and use this one instead.

Reply:
well in the editor it goes from 0 to 10 
so 
10 = 10 times damage
0 = you take zero damage from that type of damage

Reply:
yeah, so what I'm thinking now is this new component makes more sense to handle resistance and weakness. We should leave armour as only granted by clothing, and only add so they could not handle negative numbers.

DamageResistance on the other hand, could handle negative numbers which would mean a weakness. In the end it is basically the same, but I feel like that is more sane.

Reply:
I'm leaning towards Damage Weakness, because the bigger the value the Weaker it is, if it resistance should be the other way around?

Reply:
yes, it should be other way around.